packages/amplify-codegen/src/utils/loading.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const { extname } = require('path');
+
+function loadSchema(schemaPath) {
+  if (!fs.existsSync(schemaPath)) {
+    throw new Error(`Cannot find GraphQL schema file: ${schemaPath}`);
+  }
+  return fs.readFileSync(schemaPath, 'utf8');
+}
+
+function isSDLSchema(schemaPath) {
```

Comment:
Is this used anywhere?

Reply:
Depending on the refactor in upcoming PR for the top level codegen plugin, we might just remove this one.